packages/test/src/config.js

```diff
@@ -1,12 +1,13 @@
 const path = require('path');
 
 const cwd = process.cwd();
+const rootDir = path.resolve(__dirname, '../');
 
 module.exports = {
-  rootDir: path.resolve(__dirname, '../'),
-  roots: [`${cwd}/webpack/`, `${cwd}/script/`],
-  automock: true,
+  rootDir: cwd,
```

Comment:
I see it confusing, having `cwd` and `rootDir` while the `config.rootDir = cwd`.

Considre changing to:
```js
const consumerRoot = process.cwd();
const tfmTestRoot = path.resolve(__dirname, '../');

module.exports = {
  rootDir: tfmTestRoot,
  ...
};
```